azure-pipelines.yml

```diff
@@ -221,10 +221,11 @@ stages:
             yarn
             node ./scripts/retrieveDebugProxy.js
             md $(Build.SourcesDirectory)/artifacts/packages/VSCode/$(_BuildConfig)/ -ea 0
-            npx vsce@1.100.2 package -o $(Build.SourcesDirectory)/artifacts/packages/VSCode/$(_BuildConfig)/blazorwasm-companion-$version.vsix
+            npx vsce@1.103.1 package -o $(Build.SourcesDirectory)/artifacts/packages/VSCode/$(_BuildConfig)/blazorwasm-companion-$version.vsix
```

Comment:
For posterity, this change was made because apparently version 1.100.2 of vsce outputs some mundane, non-error text on the error channel, which is probably why we didn't have "failOnStderr" on in the first place. Thankfully they fixed it in  1.103.1.